<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralUserTab</class>
 <widget class="QWidget" name="GeneralUserTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="name_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">PLACEHOLDER</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="description_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="first_name_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="last_name_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fullNameLabel">
       <property name="text">
        <string>Display Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="display_name_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="initialsLabel">
       <property name="text">
        <string>Initials:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="initials_edit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="eMailLabel">
       <property name="text">
        <string>E-mail Address:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="email_edit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="officeLocationLabel">
       <property name="text">
        <string>Office Location:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="office_edit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="telephoneNumberLabel">
       <property name="text">
        <string>Telephone Number:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="webPageAddressLabel">
       <property name="text">
        <string>Web Page Address:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="telephone_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="telephone_button">
         <property name="text">
          <string>Other...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="web_page_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="web_page_button">
         <property name="text">
          <string>Other...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="middle_name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="middle_name_label">
       <property name="text">
        <string>Middle Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
